src/applications/vaos/utils/timezone.js

```diff
@@ -1,5 +1,5 @@
+import { format, formatInTimeZone } from 'date-fns-tz';
 import timezones from './timezones.json';
-import moment from '../lib/moment-tz';
```

Comment:
Hmm I think this PR gets rid of all references to `lib/moment-tz` so we can remove that file?